resources/lrsql/config/config.edn

```diff
@@ -1,75 +1,99 @@
-                     :db-name "example.sqlite.db"}
-   :test-sqlite-mem {:db-type "sqlite"
-                     :db-name ":memory:"}
-   :test-postgres   {:db-type         "postgres"
-                     :db-name         "lrsql_db"
-                     :db-host         "0.0.0.0"
-                     :db-port         5432
-                     :db-user         "lrsql_user"
-                     :db-password     "lrsql_password"
-                     ;; Schemas are commented out for testing (since all tests
-                     ;; happen in the default `public` schema). We can uncomment
-                     ;; these properties for manual schema testing.
-                     ;; :db-schema     "lrsql"
-                     ;; :db-properties "currentSchema=lrsql"
+ {;; Test/Dev
+  :test-sqlite     {:db-type "sqlite"
+                    :db-name "example.sqlite.db"}
+  :test-sqlite-mem {:db-type "sqlite"
+                    :db-name ":memory:"}
+  :test-postgres   {:db-type     "postgres"
+                    :db-name     "lrsql_db"
+                    :db-host     "0.0.0.0"
+                    :db-port     5432
+                    :db-user     "lrsql_user"
+                    :db-password "lrsql_password"
+                    ;; Schemas are commented out for testing (since all tests
+                    ;; happen in the default `public` schema). We can uncomment
+                    ;; these properties for manual schema testing.
+                    ;; :db-schema     "lrsql"
+                    ;; :db-properties "currentSchema=lrsql"
 
-                     ;; Testing Only! Specify the version used with testcontainers
-                     :test-db-version #or [#env LRSQL_TEST_DB_VERSION "11"]}
-   :test-oidc       {:db-type "sqlite"
-                     :db-name ":memory:"}
+                    ;; Testing Only! Specify the version used with testcontainers
+                    :test-db-version #or [#env LRSQL_TEST_DB_VERSION "11"]}
+  :test-mariadb {:db-type       "mariadb"
+                 :db-name       "lrsql_db"
+                 :db-host       "0.0.0.0"
+                 :db-port       3306
+                 :db-user       "lrsql_user"
+                 :db-password   "lrsql_password"
+                 :db-properties "allowMultiQueries=true"
 
-   ;; Production
-   :prod-sqlite-mem #include "prod/sqlite_mem/database.edn"
-   :prod-sqlite     #include "prod/sqlite/database.edn"
-   :prod-postgres   #include "prod/postgres/database.edn"}
+                 ;; Testing Only! Specify the version used with testcontainers
+                 :test-db-version #or [#env LRSQL_TEST_DB_VERSION "11.7.2"]}
+  :test-oidc {:db-type "sqlite"
+              :db-name ":memory:"}
+
+  ;; Production
+  :prod-sqlite-mem #include "prod/sqlite_mem/database.edn"
+  :prod-sqlite     #include "prod/sqlite/database.edn"
+  :prod-postgres   #include "prod/postgres/database.edn"
+  :prod-maria      #include "prod/maria/database.edn"
```

Comment:
assuming you're going to make `:prod-maria` consistent with `:test-mariadb`

Reply:
brought in line